frontend/src/components/worksheets/BundleDetail/BundleFieldTable/BundleFieldRow.jsx

```diff
@@ -27,8 +27,45 @@ class BundleFieldRow extends React.Component {
         super(props);
     }
 
+    checkHideRow() {
+        const field = this.props.field || {};
+        const value = this.props.value || field?.value;
+        if (!field.editable) {
+            if (!value) {
+                return true;
+            }
+            if (field.type === 'list' && (!value.length || !value[0])) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    getTitle() {
+        const field = this.props.field || {};
+        const description = this.props.description || field.description;
+        if (!description) {
+            return;
+        }
+        const name = field.name;
+        if (!name) {
+            return description;
+        }
+        const title = (
+            <>
+                {description}
+                <br />
+                <br />
+                Search Key: '{name}'
```

Comment:
I'm not sure 'search key' is that intuitive, because it's not just used for search, and it's not clear how you'd use this to search in the UI.  Maybe we can just put it in brackets at the end?  I feel like it should be something that shouldn't draw attention because most people won't need it, but it should be easy for experts who already know about these fields to find it.

Reply:
Brackets makes sense. If we're going to go that route tho, I think it would be better to have the field names at the beginning of the description instead of at the end tho. If we place them at the end, even experts may not realize it's there.

![Screen Shot 2022-08-04 at 5 27 34 PM](https://user-images.githubusercontent.com/25855750/182977289-1101bcdd-3a08-4c3f-9e73-b5646c4fcf52.png)

Lmk your thoughts

Reply:
@leilenah Should we instead put the field key names in parenthesis and at the end, for readability reasons? So "Amount of memory ..... (request_memory)". I think that starting the tooltip with something that looks like code and isn't English (`[request_memory]`) may be intimidating / not look very nice from the user's perspective.

Reply:
@epicfaace fair, can do